### DISCUSSION TITLE
Call for Showcase links

### DISCUSSION THREAD

**Discussion by benallfree** [pockethost/pockethost#180] [Q&A]
Some users (and me) are interested in showcasing PocketHost projects.

* https://github.com/benallfree/pockethost/discussions/166
* https://github.com/benallfree/pockethost/discussions/152#discussioncomment-4575050
* https://github.com/benallfree/pockethost/discussions/152#discussioncomment-4692255
* https://pbrealtimeblog.vercel.app/ from #10 

If you have a project you'd like showcased, please leave a link in this ticket.

**Comment by osmely** [comment]
[ProfileHub](https://www.profilehub.org)
Platform for the management of public profiles.

This is a project in progress. There is still a way to go yet, but also an excellent way to learn. Thanks to @benallfree and [pockethost](https://pockethost.io)

**Reply by Hyperclaw79** [reply]
Hey @osmely wanted to check this out but it is redirecting me to some Handyman service....

**Comment by TechStudent10** [comment]
[Proton Designer](https://github.com/ProtonDesigner/core)

Simple yet powerful application creation. This has been my hobby project for the past couple of months (still in prerelease) and if not for PocketBase and PocketHost I don't think I would even have a db for this. Thanks if you check out the app and thanks to the developers of both PocketBase and PocketHost. Really awesome stuff happening here.

**Comment by benallfree** [comment]
@TechStudent10 Does Proton Designer have a live URL?

**Comment by TechStudent10** [comment]
> @TechStudent10 Does Proton Designer have a live URL?

@benallfree not yet. Plan to work on the site this weekend. I just discovered Astro and now want to use it everywhere haha.

**Comment by gitatmanu** [comment]
https://estadodelmce.netlify.app/

**Reply by benallfree** [reply]
Hey, me gusta! Grácias!

Can you please provide a short English description here?

**Comment by Mellbin** [comment]
https://web.mlbn.se ( soon to be https://sandhack.se) when development is compelted.

**Reply by benallfree** [reply]
Very nice!

**Comment by Hyperclaw79** [comment]
[HT Developer Portfolio](https://ht-dev.vercel.app/)

I was creating a Developer Portfolio and chose to go with PocketBase as the DB since I wanted something very light.
I wanted to host my local DB online and PocketHost was the drop in solution for me!

And here is the source in case anyone is interested:
https://github.com/Hyperclaw79/ht.dev

**Comment by benallfree** [comment]
@Hyperclaw79 @Mellbin @TechStudent10 @gitatmanu Please see https://github.com/pockethost/pockethost/discussions/337 to get your showcase page added.

**Reply by Hyperclaw79** [reply]
And done:
https://github.com/pockethost/pockethost/pull/344

**Comment by humbertomorais** [comment]
I subscribed to the Starter Plan and it was not updated on the Dashboard. My email is [humberto.silva@ifma.edu.br](mailto:humberto.silva@ifma.edu.br). Could you help me?
